package.json

```diff
@@ -29,15 +30,14 @@
     "cors": "2.8.5",
     "dayjs": "1.8.14",
     "envinfo": "7.3.1",
-    "express": "4.16.4",
+    "express": "4.17.1",
     "handlebars": "4.1.2",
-    "http-errors": "1.7.3",
     "js-yaml": "3.13.1",
     "jsonwebtoken": "8.5.1",
     "kleur": "3.0.3",
-    "lodash": "4.17.11",
+    "lodash": "4.17.14",
     "lunr-mutable-indexes": "2.3.2",
-    "marked": "0.7.0",
+    "marked": "0.6.3",
```

Comment:
Is it any problem with this dep? A downgrade is done

Reply:
Good catch